cli/args/flags.rs

```diff
@@ -818,49 +818,10 @@ fn clap_root() -> Command {
 }
 
 fn bench_subcommand() -> Command {
-  Command::new("bench").defer(|cmd| {
-    runtime_args(cmd, true, false)
-      .arg(check_arg(true))
-      .arg(
-        Arg::new("json")
-          .long("json")
-          .action(ArgAction::SetTrue)
-          .help("UNSTABLE: Output benchmark result in JSON format"),
-      )
-      .arg(
-        Arg::new("ignore")
-          .long("ignore")
-          .num_args(1..)
-          .use_value_delimiter(true)
-          .require_equals(true)
-          .help("Ignore files")
-          .value_parser(value_parser!(PathBuf)),
-      )
-      .arg(
-        Arg::new("filter")
-          .long("filter")
-          .allow_hyphen_values(true)
-          .help("Run benchmarks with this string or pattern in the bench name"),
-      )
-      .arg(
-        Arg::new("files")
-          .help("List of file names to run")
-          .num_args(..)
-          .value_parser(value_parser!(PathBuf))
-          .action(ArgAction::Append),
-      )
-      .arg(
-        Arg::new("no-run")
-          .long("no-run")
-          .help("Cache bench modules, but don't run benchmarks")
-          .action(ArgAction::SetTrue),
-      )
-      .arg(watch_arg(false))
-      .arg(no_clear_screen_arg())
-      .arg(script_arg().last(true))
-      .about("Run benchmarks")
-      .long_about(
-        "Run benchmarks using Deno's built-in bench tool.
+  Command::new("bench")
+    .about("Run benchmarks")
+    .long_about(
```

Comment:
The difference might be hard to see, but all I did was move it outside of the block.